cmd/minishift/cmd/update.go

```diff
@@ -89,16 +89,16 @@ func runUpdate(cmd *cobra.Command, args []string) {
 				atexit.ExitWithMessage(1, fmt.Sprintf("Update failed: %s", err))
 			}
 
-			fmt.Printf("\nUpdated successfully to minishift v%s.\n", latestVersion)
+			fmt.Printf("\nUpdated successfully to Minishift version %s.\n", latestVersion)
 
 			markerData := UpdateMarker{InstallAddon: false, PreviousVersion: version.GetMinishiftVersion()}
 
-			fmt.Print("\nDo you want to update default addons? [y/N]: ")
+			fmt.Print("\nDo you want to update the default add-ons? [y/N]: ")
 			fmt.Scanln(&confirm)
 
 			if strings.ToLower(confirm) == "y" {
 				markerData.InstallAddon = true
-				fmt.Println("Default add-ons will be updated when executing the next minishift command.")
+				fmt.Println("Default add-ons will be updated the next time you run any `minishift` command.")
```

Comment:
I think we agreed to not use backticks in the help messages. This is not markdown.